libraries/rush-lib/src/cli/actions/BaseInstallAction.ts

```diff
@@ -159,9 +132,9 @@ export abstract class BaseInstallAction extends BaseRushAction {
         selectedSubspaces = this.rushConfiguration.getSubspacesForProjects(
           new Set(installManagerOptions.filteredProjects)
         );
-      } else if (this._subspaceParameter.value) {
+      } else if (this._selectionParameters?.getTargetSubspace()) {
         // Selecting a single subspace
-        const selectedSubspace: Subspace = this.rushConfiguration.getSubspace(this._subspaceParameter.value);
+        const selectedSubspace: Subspace = this._selectionParameters?.getTargetSubspace();
```

Comment:
Can you refactor this so we can avoid calling the function twice? Instead, can you stick the result in a variable and check its value in the `if` statement?